### ISSUE TITLE
Focus resets after opening and closing settings

### ISSUE THREAD

**Issue by robintown** [vector-im/element-call#3965]
### Steps to reproduce

1. Use Tab to focus the settings button
2. Open settings with Space
3. Close settings with Escape

#### What did you expect?

Focus remains visible on the settings button

#### What happened instead?

Focus is not visible anywhere. If I press Tab again, focus resets to the elements at the start of the page.

### Operating system

NixOS 25.11

### Browser information

Firefox 150.0.2

### URL for webapp

develop.element.io

### Will you send logs?

No
